lib/openhab/dsl/items/rollershutter_item.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+require 'java'
+
+module OpenHAB
+  module DSL
+    module Items
+      #
+      # Delegator to OpenHAB Rollershutter Item
+      #
+      class RollershutterItem
+        extend Forwardable
+        include Comparable
+
+        def_delegator :@rollershutter_item, :to_s
+
+        java_import Java::OrgOpenhabCoreLibraryTypes::PercentType
+        java_import Java::OrgOpenhabCoreLibraryTypes::UpDownType
+        java_import Java::OrgOpenhabCoreLibraryTypes::StopMoveType
+
+        #
+        # Creates a new RollershutterItem
+        #
+        # @param [Java::OrgOpenhabCoreLibraryItems::RollershutterItem] rollershutter_item
+        #   The OpenHAB RollershutterItem to delegate to
+        #
+        def initialize(rollershutter_item)
+          logger.trace("Wrapping #{rollershutter_item}")
+          @rollershutter_item = rollershutter_item
+        end
+
+        #
+        # Check if the rollershutter is up
+        #
+        # @return [Boolean] true if the rollershutter is up, false otherwise
+        #
+        def up?
+          state.as(UpDownType) == UpDownType::UP
+        end
+
+        #
+        # Check if the rollershutter is down
+        #
+        # @return [Boolean] true if the rollershutter is down, false otherwise
+        #
+        def down?
+          state.as(UpDownType) == UpDownType::DOWN
+        end
+
+        #
+        # Returns the rollershutter's position
+        #
+        # @return [Java::OrgOpenhabCoreLibraryTypes::PercentType] the position of the rollershutter
+        #
+        def position
+          state.as(PercentType)
+        end
+
```

Comment:
This should support coerce too, right?
So you can do:
```RollerShutter << 10 + OtherShutter```

Reply:
Not sure how to implement this actually, since this just wraps the item, and I can't coerce a number into that. Would have to create a wrapper class for PercentType and coerce the number to that?

Reply:
Is that different than what we do in NumberItem?  I think on coere we just return a Quantity or even a BigDecimal?  Something that can then be added or subtracted, etc?

Reply:
Haven't really got the grasps on coerce, but if I just need to return two numbers that can be added could I convert self to an Integer? That wouldn't be a problem

Reply:
I've seen this elsewhere in our library too - we seem to be treating PercentType as Integer, but it is actually a DecimalType. Should we treat it as a Float?